modules/apps/commerce/commerce-product-api/src/main/java/com/liferay/commerce/product/option/CommerceOptionType.java

```diff
@@ -21,6 +21,8 @@ public interface CommerceOptionType {
 
 	public String getLabel(Locale locale);
 
+	public boolean hasValues();
```

Comment:
What is this for?

Reply:
This is to determin if the option type have values or the type does not accept them